lib/node/http.js

```diff
@@ -129,8 +130,8 @@ class HTTP extends Server {
           state: {
             tx: this.chain.db.state.tx,
             coin: this.chain.db.state.coin,
-            value: this.chain.db.state.value,
-            burned: this.chain.db.state.burned
+            value: Amount.coin(this.chain.db.state.value, true),
+            burned: Amount.coin(this.chain.db.state.burned, true)
           }
```

Comment:
The RPC interface often returns `Amount.coin` with the second argument to `true`. See bottom.

The code path for `Amount.coin` ends us in `Amount.encode` (below), where the second argument of `coin` is `num` in this context, which causes the number to be returned as a float.

https://github.com/handshake-org/hsd/blob/4cdaa33fa4483b3a6c74761479c90cb96bf9b36f/lib/ui/amount.js#L341-L345

https://github.com/handshake-org/hsd/blob/4a0ce94a390158e0d1e60838b818379403d1059e/lib/wallet/rpc.js#L505